orchagent/orch.cpp

```diff
@@ -845,19 +845,19 @@ void Orch2::doTask(Consumer &consumer)
         }
         catch (const std::invalid_argument& e)
         {
-            SWSS_LOG_ERROR("Parse error: %s", e.what());
+            SWSS_LOG_ERROR("Parse error in %s: %s", typeid(*this).name(), e.what());
         }
         catch (const std::logic_error& e)
         {
-            SWSS_LOG_ERROR("Logic error: %s", e.what());
+            SWSS_LOG_ERROR("Logic error in %s: %s", typeid(*this).name(), e.what());
         }
         catch (const std::exception& e)
         {
-            SWSS_LOG_ERROR("Exception was catched in the request parser: %s", e.what());
+            SWSS_LOG_ERROR("Exception was caught in the request parser in %s: %s", typeid(*this).name(), e.what());
```

Comment:
@DavidZagury maybe it is better to use `__PRETTY_FUNCTION__` ? If it works, suggest to use this instead of `typeid`
P.S: you can try to inject it directly into `SWSS_LOG_*` plugin

Reply:
When I changed the log to:
`SWSS_LOG_ERROR("%s: Logic error in %s: %s", __PRETTY_FUNCTION__, typeid(*this).name(), e.what());
`
The log produced was:
`ERR swss#orchagent: :- doTask: virtual void Orch2::doTask(Consumer&): Logic error in 15VxlanTunnelOrch: map::at
`
So it didn't helped much